### ISSUE TITLE
[BUG] Alert: missing announcement on open when screen reader is being used  

### ISSUE THREAD

**Issue by Fuzzy3** [kirbydesign/designsystem#3679]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
There is no indication from the screen reader that an alert has been opened. An opening of an alert should be clearly announced in a such a way that the user knows that he/she should take some action.    

## Describe how to reproduce the bug

1. Open the cookbook and go to alert
2. Turn on a screen reader
3. Navigate to the "Show Alert" button and click it
4. Notice how the alert was opened but without an indication from the screen reader that it was opened.

## Add any screenshots

<img width="1356" alt="image" src="https://github.com/user-attachments/assets/7ab7731c-140d-437e-91cd-cf52693ccfd5">

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.
